groovy/gradle/manifest.mf

```diff
@@ -3,5 +3,5 @@ AutoUpdate-Show-In-Client: false
 OpenIDE-Module: org.netbeans.modules.gradle/2
 OpenIDE-Module-Layer: org/netbeans/modules/gradle/layer.xml
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/gradle/Bundle.properties
-OpenIDE-Module-Specification-Version: 2.1
+OpenIDE-Module-Specification-Version: 2.1.1
```

Comment:
For the integration into master I agree with this version, but for integration into release120 it should be 2.0.1.

Reply:
Thank you for the review!
1. I'm aware of the different versions on master and on release120 branch and I'm going to do my merges according to that.
2. Agree squashing this for merge is planned.
3. API change documentation is coming soon, as we are fine with the other parts.

Thanks again!